quickwit-doc-mapper/src/default_doc_mapper/default_mapper.rs

```diff
@@ -354,15 +377,18 @@ pub struct DefaultDocMapper {
     pub timestamp_field_name: Option<String>,
     /// Sort field name and order.
     pub sort_by: SortBy,
-    /// List of field mappings which defines how a json field is mapped to index fields.
-    pub field_mappings: FieldMappingEntry,
+    /// Root field mapping.
```

Comment:
```suggestion
    /// Root field mapping. It has an empty name so that we don't prefix all field names with it. See [FieldMappingEntry::root]
```
unsure if the link works :D